pages/common/R.md

```diff
@@ -0,0 +1,32 @@
+# R
+
+> R language interpreter.
+> More information: <https://www.r-project.org>.
+
+- Start an R interactive shell (REPL):
+
+`R`
+
+- Check R version:
+
+`R --version`
+
+- Start R in vanilla mode (i.e. blank session that doesn't save workspace at the end):
+
+`R --vanilla`
+
+- Execute a file:
+
+`R -f {{file.R}}`
+
+- Execute R expression and exit:
+
+`R -e {{expr}}`
+
+- Run R through a debugger:
+
+`R -d {{debugger}}`
+
+- Check R packages from package sources:
+
+`R CMD check {{package_source}}`
```

Comment:
Is this a file/directory path, or something else?

Reply:
It is either a directory or a `tar` archive. The official `man` documentation reads:

> Usage: R CMD check [options] pkgs
>
> Check R packages from package sources, which can be directories or package 'tar' archives with extension '.tar.gz', '.tar.bz2', '.tar.xz' or '.tgz'.

Reply:
Also, to create the mentioned `tar` archive, one would need to use `R CMD build {{/path/to/package_source}}`. This would create ther `tar` archive to be used for R CMD check.

Alternatively, it is possible to run `R CMD check` directly on the directory without first creating the archive. As this command can be run directly, I am not sure if we should include the `R CMD build {{/path/to/package_source}}` in this page.

Reply:
Ah, I see. What about this then?
```suggestion
`R CMD check {{path/to/package_source}}`
```

Reply:
Seems good!